<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>494</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>605</width>
    <height>494</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>605</width>
    <height>494</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;abc&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>531</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbImage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>521</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Chọn hình ảnh nhận dạng</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChosseFolder">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Chọn thư mục hình ảnh </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGetRealTimeFace">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Chụp Ảnh trực tiếp </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLuuThongTin">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Lưu thông tin</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
